spec/actions/app_spec.cr

```diff
@@ -1,3 +1,4 @@
+require "../../src/actions"
```

Comment:
Why add a require here, instead of somewhere within *spec_helper*?

Reply:
This isn't the purpose of `spec_helper`.
We require only what we need, depending on what part we test.
Therefore, we can affirm that only this module of our app is tested, which is self-contained.

This also speed up running individual files. That's how it's done in the Crystal repo too.